front/src/applications/operationalStudies/hooks/useScenarioData.ts

```diff
@@ -322,6 +335,78 @@ const useScenarioData = (scenario: ScenarioResponse, infra: InfraWithState) => {
     [timetableItems, rollingStocks]
   );
 
+  const upsertTimetableItemsWithBroadcast = useCallback(
+    (timetableItemsToUpsert: TimetableItemWithTimetableId[]) => {
```

Comment:
nit : to be consistent with the other two functions (or the other way, be more preciser for the other 2 functions)
```suggestion
    (ids: TimetableItemWithTimetableId[]) => {
```

Reply:
The other two functions take IDs as inputs, while this one takes a full timetable item (ie. an object instead of a string).

Reply:
woops, got baited by the id at the end of the type (TGIF)